_layouts/partners/faq.html

```diff
@@ -0,0 +1,127 @@
+---
+layout: base
+---
+<header>
+  {% include skip_nav.html %}
+  {% include banner.html %}
+  {% include partners/header.html %}
+</header>
+<script>
+  function closeAllAccordions(){
+    // changes accordion button look through aria controls
+    let accordionButtons = document.getElementsByClassName('usa-accordion__button');
+    Array.from(accordionButtons).forEach(el => {
+      el.setAttribute("aria-expanded", "false");
+    });
+    // hides accordion content panels
+    let accordionContentPanels = document.getElementsByClassName('usa-accordion__content');
+    Array.from(accordionContentPanels).forEach(el => {
+      el.hidden = true;
+    })
+  };
+</script>
+<main id="main-content">
+    <section>
+        <div class="container partners-container">
+            <div class="grid-row">
+                <div class="tablet:grid-col-9 accordion-container">
+                  <div class="partners-body partners-title">{{ page.title | markdownify }}</div>
+                  <div class="partners-border"></div>
+                  <div class="section-title-row">
+                    <div id="general-section" class="section-title-left">{{ page.general_section_title | markdownify}}</div>
+                    <a class="section-title-right" href="javascript:closeAllAccordions()">Collapse all</a>
+                  </div>
+                  <div class="usa-accordion usa-accordion--bordered margin-bottom-5" aria-multiselectable="true">
```

Comment:
Couple of things I see with the accordion: 
The text for the h4s and ps don't seem to be the right font
The h4s are 14.56px when they should be 16px
The accordion in its expansed view doesn't have lines around the text